e2e/api/steps.js

```diff
@@ -289,6 +289,7 @@ module.exports = {
 
   amendPartyDetails: async (user) => {
     await apiRequest.setupTokens(user);
+    caseData = {};
```

Comment:
Is this change needed at all?

Reply:
@bhssamray yes it is, need to clear the casedata or else api tests would error out